spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -8,6 +8,38 @@ Data Cloud Spanner module link:../../spring-cloud-gcp-starters/spring-cloud-gcp-
 
 . Create a new Cloud Spanner instance named "spring-demo".
 The example application will create the database and tables if they do not already exist.
+
+You can use https://cloud.google.com/spanner/docs/emulator[Cloud Spanner emulator] to run the sample locally.
+To install the Cloud Spanner emulator:
+
+Install the Google Cloud SDK.
```

Comment:
I don't know if we need to repeat all of the steps here because it's likely they will change.
You can say, once you followed the instructions and created the emulator configuration, activate it...

Reply:
If you want, maybe one way to avoid duplication would be to link to the subsections of the emulator docs in your list of steps, like:

Instructions for using the Cloud Spanner Emulator:

1. Follow the steps to [install and start the Spanner Emulator](https://cloud.google.com/spanner/docs/emulator#installing_and_running_the_emulator)

2. Create and update the emulator configuration
....

etc.